packages/pages/src/dev/server/ssr/inPlatformPageSets.ts

```diff
@@ -55,8 +55,8 @@ export const getInPlatformPageSetDocuments = async (
 
   return await spawnPageSetCommands(
     process.stdout,
-    "yext pages visual-editor document",
-    args,
+    "yext",
+    ["pages", "visual-editor", "document", ...args],
```

Comment:
Should we fix generateTestData too? Looks like it's the same. Double check that everything still works of course with shell: true no longer being set.

Reply:
Hmm ok we can return to this later if necessary.